content/operate/kubernetes/deployment/openshift/openshift-operatorhub.md

```diff
@@ -66,11 +66,15 @@ You only need to install the SCC once, but you must not delete it.
 After the install, the OperatorHub automatically uses the constraint for Redis Enterprise node pods.
 
 {{< note >}}
-**Known Limitation** - The automatic use of the security constraint is limited. The
-Redis Enterprise must be named `rec` for the constraint to be used automatically.  **Use the cluster name `rec` when deploying with the OperatorHub.**
+If you are using the recommended RedisEnterpriseCluster name of `rec`, the SCC is automatically bound to the RedisEnterpriseCluster after install.
+
+If you choose a different name for the RedisEnterpriseCluster, or override the default service account name, you must manually bind SCC to the RedisEnterpriseCluster’s service account:
+
+  ```sh
+  oc adm policy add-scc-to-user redis-enterprise-scc-v2 \
+  system:serviceaccount:<my-project>:<rec-name>
```

Comment:
I wonder if we should be more accurate here and replace `<rec-name>` with `<rec-service-account-name`.
In principle, this is indeed a name of a service account. By default, the SA is named similarly to the REC, unless overridden via the REC's `.spec.serviceAccountName` field.